qiskit_algorithms/amplitude_estimators/iae.py

```diff
@@ -54,7 +55,9 @@ def __init__(
         alpha: float,
         confint_method: str = "beta",
         min_ratio: float = 2,
-        sampler: BaseSampler | None = None,
+        sampler: BaseSamplerV2 | None = None,
+        transpiler: Transpiler | None = None,
+        transpiler_options: dict[str, Any] | None = None,
```

Comment:
If this is left as the default None (or explicitly passed as None of course) does it work - my impression is that it would be doing a **None when calling the transpiler method passed. The tests I saw did not cover this aspect - unless I missed things.

Reply:
I am not quite sure why you resolved this. Does it not need the transpiler_options internally setting to {} like I see you did elsewhere and commented below.